src/auth/auth.service.ts

```diff
@@ -1,8 +1,11 @@
+import { ApiException } from './../shared/errors/api-exception';
```

Comment:
Codacy found an issue: ['ApiException' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5747194)